tokens/src/tokens.js

```diff
@@ -22,11 +24,24 @@ export const font = {
     bold: 700
   },
   lineHeight: {
-    smaller: 1,
-    regular: 1.5
+    target: {
+      small: 16,
+      medium: 24,
+      large: 32,
+    },
+    ratio: {
+      smaller: 1,
+      small: 1.2,
+      medium: 1.5
+    }
   }
 };
 
+export const fontMetrics = (size = 'medium', target = 'medium') => ({ theme = { font } }) => css`
```

Comment:
I'm not sure if `@nulogy/tokens` should be exporting helper functions ... We should aim to keep it as simple as possible and as close to pure json data (even though its actually not technically conferment json atm).

I'm also pretty sure that `@nulogy/tokens` doesn't actually have `styled-components` in its package json. Nor should it.

This should go someplace else: Either a utils folder of the `@nulogy/components` package, or better yet, a new shared package `@nulogy/theme` perhaps?

There is a mediaQuery helper function introduced in PR #42 that would go nicely in that package as well.